docs/api/source/conf.py

```diff
@@ -0,0 +1,114 @@
+import importlib
```

Comment:
This file controls the documentation build process that is executed during `sphinx-build`.